catalog-entities/marketplace/plugins/mcp-tools.yaml

```diff
@@ -0,0 +1,228 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/redhat-developer/rhdh-plugins/refs/heads/main/workspaces/marketplace/json-schema/plugins.json
+apiVersion: extensions.backstage.io/v1alpha1
+kind: Plugin
+metadata:
+ name: mcp-tools
+ namespace: rhdh
+ title: MCP Tools
+ annotations:
+   extensions.backstage.io/pre-installed: 'true' # For plugins contained in the RHDH image
```

Comment:
don't we have separate images for our dynamic plugins @johnmcollier  .... the comment implies the plugins come from the core image

Reply:
D’oh, yes. I meant to change that after Ben’s feedback but forgot